frontend/demo/src/model/Channel.ts

```diff
@@ -0,0 +1,56 @@
+export interface Channel {
+  name: string;
+  source: string;
+  source_channel_id: string;
```

Comment:
we want to use camelCase in out interfaces, that's why we have the mapper, to map from backend code (snake_case) to frontend code (camelCase)